### DISCUSSION TITLE
Having authentication issues

### DISCUSSION THREAD

**Discussion by Bussyslim** [community/community#49221] [New to GitHub]
hi 
i cannot  push from git to my github. i tried generating access token from github still my access is been denied

**Comment by jge162** [comment]
Hello @Bussyslim Without further details here are some things you can double check 

- Incorrect remote URL:
- Incorrect credentials:
- Firewall or proxy issues:
- Network issues 
- Incorrect permissions

This is just a few things to verify. Hopefully you can catch the issue.

**Comment by SMelidoni** [comment]
Hi there @Bussyslim 👋🏽

There could be several reasons why you're unable to push to your GitHub repository even after generating an access token. Here are some troubleshooting steps you can try:

1. Check if you're using the correct access token: Make sure that you're using the correct access token when pushing to your GitHub repository. You can create a personal access token from your GitHub account settings and use it as your password when pushing.

2. Verify your repository URL: Ensure that the remote URL of your repository is correct. You can check this by running the following command in your local Git repository:

`git remote -v`

This will show you the URLs of the remotes associated with your local Git repository.

3. Check if you have the necessary permissions: Verify that you have the necessary permissions to push to the repository. If the repository is owned by an organisation, make sure that you have the necessary access level to push to the repository.

4. Try pushing with HTTPS instead of SSH: If you're using SSH to push to your GitHub repository, try using HTTPS instead. Sometimes, SSH can be blocked by firewalls, so switching to HTTPS might resolve the issue.

5. Check your network connection: Ensure that you have a stable network connection. A weak or unstable network connection can cause issues when pushing to GitHub.

If none of these steps resolve your issue, you might want to try reaching out to GitHub support for further assistance.

Hope this helps!
